packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -109,7 +109,7 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
    * an unfortunate side effect is that users will not be able to tab out of the focus zone
    * and have to hit escape or some other key.
    */
-  allowTabKey?: boolean;
+  tabPermission?: FocusZoneTabbableElements;
```

Comment:
Let's go ahead and deprecate allowTabKey instead of removing it. Remove all references to it, put at @deeprecated in the jsdoc comment, and add a _warnDeprecations for that prop referenceing tabPermission

Reply:
Consider updating the variable name, maybe `handleTabKey`